horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -168,6 +168,10 @@
     targ_proj, targ_axes, npix, s, e, ll, ...
     keep_precision, pixel_contrib_name)
 
+if numel(targ_proj) > 1
+    error('HORACE:cut:not_implemented', 'Cannot cut sym and return pixels')
+end
+
```

Comment:
I would add words "not implemented yet" to the error message but hopefully not for long so this way ok too. 

You need to add '-nopix' to tests for cut_sym for tests to pas